x-pack/legacy/plugins/ml/server/models/calendar/event_manager.ts

```diff
@@ -0,0 +1,94 @@
+/*
```

Comment:
it's strange that `calenadar_manager` has been marked as a js -> ts rename by git, but this file hasn't.
it would be useful to try to retain the git history

Reply:
yeah, I'm not sure why that's the case. I'll look into it.

Reply:
it looks ok now. t shows it as a moved file. one of your subsequent comments must have made the difference